forge-gui/res/cardsfolder/upcoming/lay_down_arms.txt

```diff
@@ -0,0 +1,10 @@
+Name:Lay Down Arms
+ManaCost:W
+Types:Sorcery
+A:SP$ ChangeZone | ValidTgts$ Creature.cmcLEX | TgtPrompt$ Select target creature with mana value less or equal to the number of plains you control. | Origin$ Battlefield | Destination$ Exile | RememberLKI$ True | SubAbility$ DBGainLife | SpellDescription$ Exile target creature with mana value less than or equal to the number of Plains you control. Its controller gains 3 life.
+SVar:DBGainLife:DB$ GainLife | LifeAmount$ 3 | Defined$ RememberedController | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:X:Count$Valid Plains.YouCtrl
+SVar:BuffedBy:Plains
+DeckHas:Ability$LifeGain
```

Comment:
Hmm... I don't think we want this... the AI probably isn't going to use this to exile its own creature and gain life... yet? @Agetian thoughts? I know Swords to Plowshares and similar currently don't have this tag.

Reply:
removed it for now.